index.html

```diff
@@ -72,9 +74,20 @@
 								that this has already happened.
 							</div>
 						</div>
+						<div class="tabs-item" data-tab="5">
+							<div class="tabs-item-title">Quote 5</div>
+							<div class="tabs-item-description">
+								What we've got here is failure to communicate.
+							</div>
+						</div>
+            			<div class="tabs-item" data-tab="6">
```

Comment:
Not sure what's going on with your tabs comments, but hope everything is okay :) Your code is good, though!